fiaas_deploy_daemon/__init__.py

```diff
@@ -110,7 +110,7 @@ def init_k8s_client(config: Configuration, log: logging.Logger):
             k8s_config.use_in_cluster_config()
         except IOError as e:
             if not config.client_cert:
-                log.warn("No apiserver auth config was specified, and in-cluster config could not be set up: " + str(e))
+                log.warning("No apiserver auth config was specified, and in-cluster config could not be set up: " + str(e))
```

Comment:
Since there's a `%s` in the log.warning below I guess the `str(e)` here is probably unsafe?

Reply:
fixed